.circleci/base-generated-config.yaml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+
+jobs:
+  test-example:
+    parameters:
+      example-app-name:
+        type: string
+      path:
+        type: string
+    machine:
+      image: ubuntu-2004:current
+      docker_layer_caching: true
+    environment:
+      BUILDING_BLOCK: digital-registries
+      TESTED_APPLICATION: << parameters.example-app-name >> 
+      TEST_RESULT_REPORT_FILENAME: example_result.message
+    steps:
+      - checkout
+      - run:
+          name: Execute tests
+          command: |
+            # Execute application test_entrypoint.sh in background. 
+            chmod u+x <<parameters.path >> test_entrypoint.sh &
+            cd <<parameters.path >> 
+            ./test_entrypoint.sh &
```

Comment:
I think a better approach rather than running this script in the background (&) would be just to run it normally and use `docker-compose -d ` in the test_entrypoint.sh script. That way we will avoid all kinds of races coming up from the fact the we immediately move on from executing this script